source/_integrations/slack.markdown

```diff
@@ -53,6 +54,12 @@ The added `notify` service will be named after the chat server the app is instal
 
 {% include integrations/config_flow.md %}
 
+## Integration Entities
+
+One number entity will be created:
+
+- **Do Not Disturb**: Set Do Not Disturb period. This starts immediately when set and decreases until depleted.
```

Comment:
Number entities are not allowed to decrease automatically (they are not countdown timers).